src/EA.Weee.Web/Controllers/OrganisationRegistrationController.cs

```diff
@@ -229,6 +234,88 @@ public async Task<ViewResult> JoinOrganisationConfirmation(Guid organisationId,
             }
         }
 
+        [HttpGet]
+        public ActionResult Type(string searchedText)
```

Comment:
I don't think we need the search text in this one as we will go back to a different screen. Probably need to adjust this over the coming days as the screen order might change, so for now probably can just remove the parameter.